docs/Getting Started/Example Apps/index.md

```diff
@@ -14,6 +14,28 @@ The example apps implement soft buttons, template text and images, a main menu a
 To connect the example app to [Manticore](https://smartdevicelink.com/resources/manticore/) or another emulator, make sure you are on the `TCP Debug` tab and type in the IP address and port, then press "Connect". The button will turn green when you are connected.
 
 To connect the example app to production or debug hardware, make sure you are on the `iAP` tab and press "Connect". The button will turn green when you are connected.
+
+## Troubleshooting
+### TCP Debug Transport
+1.  Make sure the correct IP address and port number is set in the `SDLLifecycleConfiguration`.
+2.  Make sure the device and SDL Core are on the same network.
+3.  If running SDL Core on a virtual machine, and you are using port forwarding to connect your device to the virtual machine, the IP address should be the IP address of your machine hosting the VM, not the IP address of the VM. The port number will be `12345`.
+4.  Make sure there is no firewall blocking the incoming port `12345` on the machine or VM running SDL Core. Also make sure your firewall allows that outgoing port.
+5.  When the app on the device is backgrounded, the app will be unable to communicate with SDL Core. This will work on IAP connections.
```

Comment:
```suggestion
5.  Due to iOS background restrictions, when the app on the device is backgrounded, the app will be unable to communicate with the SDL emulator. This will work on IAP connections.
```